app/src/main/AndroidManifest.xml

```diff
@@ -2,6 +2,7 @@
     xmlns:tools="http://schemas.android.com/tools"
     package="org.jellyfin.mobile">
 
+    <uses-permission android:name="android.permission.ACCESS_WIFI_STATE" />
```

Comment:
Was this permission required for discovery to work? If so I'll add it to the apiclient for the next release.

Reply:
Yep, is required, also is already documented in the source code by ... you (?) https://github.com/jellyfin/jellyfin-apiclient-java/blame/1565f286a3b676cdf85093b0ee99c4b940b99660/android/src/main/kotlin/org/jellyfin/apiclient/discovery/AndroidBroadcastAddressesProvider.kt#L18

Reply:
It's funny how I knew it was required but forgot to actually add it to the manifest lol